Doc/library/__future__.rst

```diff
@@ -66,7 +66,7 @@ language using this mechanism:
 +------------------+-------------+--------------+---------------------------------------------+
 | annotations      | 3.7.0b1     | Never [1]_   | :pep:`563`:                                 |
 |                  |             |              | *Postponed evaluation of annotations*,      |
-|                  |             |              | :pep:`649`: *Deferred evalutation of        |
+|                  |             |              | :pep:`649`: *Deferred evaluation of        |
```

Comment:
Try to fix sphinx formatting: add extra space after "evaluation of".  I think it should fix CI failures.

See https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#rst-tables

Reply:
Ok, thank you for the help

Reply:
Next time you might want to look on https://devguide.python.org/

E.g. documentation-related stuff: https://devguide.python.org/documentation/ (including rst markup, howto build docs, etc)